processing/src/main/java/io/druid/segment/data/ListIndexed.java

```diff
@@ -67,4 +69,10 @@ public int indexOf(T value)
   {
     return baseList.iterator();
   }
+
+  @Override
+  public void inspectRuntimeShape(RuntimeShapeInspector inspector)
+  {
+    inspector.visit("baseList", baseList);
```

Comment:
It's not used in `get()` (the only `@CalledFromHotLoop` method in `Indexed`), so it shouldn't be visited in `inspectRuntimeShape()`